src/settings/_spacing.scss

```diff
@@ -1,88 +1,80 @@
-// Single points on spacing scale
-$govuk-spacing-scale-1: 5px;
-$govuk-spacing-scale-2: 10px;
-$govuk-spacing-scale-3: 15px;
-$govuk-spacing-scale-4: 20px;
-$govuk-spacing-scale-5: 25px;
-$govuk-spacing-scale-6: 30px;
-$govuk-spacing-scale-7: 40px;
-$govuk-spacing-scale-8: 50px;
-$govuk-spacing-scale-9: 60px;
+////
+/// @group settings/spacing
+////
 
-// Responsive spacing maps
-//
-// These definitions are used to generate responsive spacing that adapts
-// according to the breakpoints (see 'helpers/spacing'). These maps should be
-// used wherever possible to standardise responsive spacing.
+/// Single point spacing variables. Access using `govuk-spacing()`
+/// (see `helpers/spacing`).
+///
+/// @type Map
+/// @access private
 
-// You can define different behaviour on tablet and desktop. The 'null'
-// breakpoint is for mobile.
+$govuk-spacing-points: (
+  0: 0,
```

Comment:
in there any point in having 0 in teh scale, since we've used 0 where needed instead of `govuk-spacing(0)?
```
.govuk-header--full-width {
   padding: 0 govuk-spacing(3);
}
```

Reply:
Yes, because it means there's an override generated for it, so you can remove margins.